mqtt_client/include/mqtt_client/MqttClient.ros2.hpp

```diff
@@ -71,6 +73,9 @@ class MqttClient : public rclcpp::Node,
   explicit MqttClient(const rclcpp::NodeOptions& options);
 
  protected:
+   class Ros2MqttInterface;
+   class Mqtt2RosInterface;
```

Comment:
forward declare as struct, as implemented below?

```suggestion
   struct Ros2MqttInterface;
   struct Mqtt2RosInterface;
```